packages/tools/generator-pie-component/src/app/templates/pie-placeholder.__stories__.ts

```diff
@@ -31,11 +31,8 @@ export default <%= componentNameCamelCase %>StoryMeta;
 
 // TODO: remove the eslint-disable rule when props are added
 // eslint-disable-next-line no-empty-pattern
-const Template = ({ <% if (needsRTL) { %>dir<% } %> }: <%= componentName %>Props): TemplateResult => html`
+const Template = ({<% if (needsRTL) {%> dir <% }%>}: <%= componentName %>Props) => html`
```

Comment:
Previously the generated text was `{  }` or `{ dir }`, now it's `{}` or `{ dir }`.